scripts/travis-tests

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env node
+
+const { execFileSync, spawnSync } = require('child_process')
+
+const run = (command, args) => {
+  const { status } = spawnSync(command, args, { stdio: 'inherit' })
+  if (status !== 0) {
+    process.exit(status)
+  }
+}
+
+const getFiles = () =>
+  execFileSync(
+    'git',
+    [
+      'diff',
+      '--diff-filter=AM',
+      '--ignore-submodules',
+      '--name-only',
+      'master',
+    ],
+    { encoding: 'utf8' }
+  )
```

Comment:
I believe (https://manpage.me/?q=git#end) that `git diff` is intended for human use, for scripts it's preferable to use `git diff-{files,index,tree}` (as I suggested in the issue related to this PR.

Reply:
fix.